File: 0119.cpp
#include <vector>

using namespace std;

class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<vector<int>> tria;
        tria.push_back({1});
        for(int i = 0; i < rowIndex; ++i) {
            vector<int> layer;
            layer.push_back({1});
            for(int j = 0; j < tria[i].size()-1; ++j) {
                layer.push_back({tria[i][j]+tria[i][j+1]});
            }
            layer.push_back({1});
            tria.push_back(layer);
        }
        return tria[rowIndex];
    }
};

File: 0119.py
class Solution:
    def getRow(self, rowIndex: int):
        tria = [[] for i in range(rowIndex+1)]
        tria[0] = [1]
        for i in range(rowIndex):
            tria[i+1].append(1)
            for j in range(len(tria[i])-1):
                tria[i+1].append(tria[i][j]+tria[i][j+1])
            tria[i+1].append(1)
        return tria[rowIndex]